Sources/Core/Formatter/Formatter.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Formatter.swift
+//  SwiftPrettyPrint
+//
+//  Created by Yusuke Hosonuma on 2020/02/26.
+//
+
+protocol Formatter {
```

Comment:
This word is general too, so it should be in Debug's name-space.

Reply:
Swift is not supported nested protocol currently :(
https://stackoverflow.com/questions/36066548/swift-protocol-nested-in-a-class

I was quite worried, I renamed to `PrettyFormatter` from `Formatter`. ( 7a792f4 )

And renamed to follows:

- `SimpleFormatter` -> `SinglelineFormatter`
- `PrettyFormatter` -> `MultilineFormatter`

What do you think?

Reply:
Oh... right!

it seems nice!
> I renamed to PrettyFormatter from Formatter. ( 7a792f4 )
> SimpleFormatter -> SinglelineFormatter
> PrettyFormatter -> MultilineFormatter